app/controllers/admins_controller.rb

```diff
@@ -1,4 +1,6 @@
 class AdminsController < ApplicationController
+  before_action :authenticate_admin!
```

Comment:
Maybe we can move the `before_action: authenticate_admin!` in the `ApplicationController`, wdyt?